src/perform-bundle/deployer.ts

```diff
@@ -0,0 +1,18 @@
+// perform-bundle/deployer.ts
+
+import { exec } from 'node:child_process';
+import { promisify } from 'node:util';
+import { debug } from 'debug';
+
+const execAsync = promisify(exec);
+
+const log = debug('perform-bundle:deployer');
+
+export default async function (): Promise<void> {
+  log('Install deployer');
+  await execAsync('npm install @checkdigit/deployer');
+  log('Install deployer complete');
+  log('Execute deployer');
+  await execAsync('export AWS_REGION=us-east-1 && npx deploy stage:lambda');
```

Comment:
since `deployer` is internal, perhaps we should call `builder` directly instead?

Reply:
As the projects don't have a handler.mts file we're relying on deployer to create the handler file first which esbuild then follows as a bundle. 
I think it is important this action tests the bundle process as used for deployment and size using the entry point we'd use in lambda. 
Results are significantly different if I just use the index.ts as the entry point.

Reply:
yeah I guess we can leave it as-is for now.  Just weird to have a public repo depending on a private module.